packages/pangea-node-sdk/src/services/vault.ts

```diff
@@ -698,7 +698,6 @@ class VaultService extends BaseService {
    *   "pvi_p6g5i3gtbvqvc3u6zugab6qs6r63tqf5",
    *   "lJkk0gCLux+Q+rPNqLPEYw=="
    *   "FfWuT2Mq/+cxa7wIugfhzi7ktZxVf926idJNgBDCysF/knY9B7M6wxqHMMPDEBs86D8OsEGuED21y3J7IGOpCQ==",
```

Comment:
Shouldn't we remove this line too?

Reply:
It's the third argument, `signature`, no?

Reply:
My bad, I saw a single `*` and thought it was an empty line, but it was github was splitting the line.